test/v1/examples.test.js

```diff
@@ -17,421 +17,446 @@
+    session.run('RETURN 1').then(() => {
+      session.close();
+    }).catch(error => {
     });
   });
 
-  it('should be able to iterate results', function(done) {
-    var session = driverGlobal.session();
-    session
-      .run( "CREATE (weapon:Weapon {name: {name}})", {name: "Sword in the stone"} )
-      .then(function() {
-    // tag::result-traversal[]
-      var searchTerm = "Sword";
-      session
-        .run( "MATCH (weapon:Weapon) WHERE weapon.name CONTAINS {term} RETURN weapon.name", {term: searchTerm} )
-        .subscribe({
-          onNext: function(record) {
-            console.log("" + record.get("weapon.name"));
-          },
-          onCompleted: function() {
-            session.close();
-          },
-          onError: function(error) {
-            console.log(error);
-          }
-        });
-    // end::result-traversal[]
-    });
+  it('config unencrypted example', done => {
+    const user = 'neo4j';
+    const password = 'neo4j';
+
+    // tag::config-unencrypted[]
+    const driver = neo4j.driver('bolt://localhost:7687', neo4j.auth.basic(user, password),
+      {
+        encrypted: 'ENCRYPTION_OFF'
+      }
+    );
+    // end::config-unencrypted[]
 
-    // Then
-    testResultPromise.then(function(loggedMsg){
-      expect(loggedMsg).toBe("Sword in the stone");
+    driver.onCompleted = () => {
       done();
+    };
+
+    const session = driver.session();
+    session.run('RETURN 1').then(() => {
+      session.close();
     });
   });
 
-  it('should be able to access records', function(done) {
-    var session = driverGlobal.session();
-    session
-      .run( "CREATE (weapon:Weapon {name: {name}, owner: {owner}, material: {material}, size: {size}})",
-          {name: "Sword in the stone", owner: "Arthur", material: "Stone", size: "Huge"})
-      .then(function() {
-      // tag::access-record[]
-        var searchTerm = "Arthur";
-        session
-          .run( "MATCH (weapon:Weapon) WHERE weapon.owner CONTAINS {term} RETURN weapon.name, weapon.material, weapon.size",
-              {term: searchTerm} )
-          .subscribe({
-            onNext: function(record) {
-              var sword = [];
-              record.forEach(function(value, key)
-              {
-                sword.push(key + ": " + value);
-              });
-              console.log(sword);
-            },
-            onCompleted: function() {
-              session.close();
-            },
-            onError: function(error) {
-              console.log(error);
-            }
-          });
-      // end::access-record[]
-      });
+  it('custom auth example', () => {
+    const principal = 'principal';
+    const credentials = 'credentials';
+    const realm = 'realm';
+    const scheme = 'scheme';
+    const parameters = {};
+
+    // tag::custom-auth[]
+    const driver = neo4j.driver(
+      'bolt://localhost:7687',
+      neo4j.auth.custom(principal, credentials, realm, scheme, parameters)
```

Comment:
Do me missing a test here?
We could still use `principle=neo4j`, `credentials=neo4j`, `real=local`, `scheme=basic` to use custom auth to pass basic auth to the server.